packages/docs/src/styles/_sections.scss

```diff
@@ -263,7 +263,6 @@
 .docs-overlay-example-transition {
   $overlay-example-width: $pt-grid-size * 40;
 
-  position: absolute;
```

Comment:
No longer necessary now that we manage the position mode internally (via `.pt-overlay-content`).